e2e/wdio/test/pageobjects/workbox.page.js

```diff
@@ -0,0 +1,93 @@
+const Page = require('./page');
+
+class WorkboxPage extends Page {
+  open() {
+    return super.open('/sitecore/shell/Applications/Workbox.aspx?sc_bw=1');
+  }
+
+  async publishContentItems() {
+    this.open();
+    await browser.switchToParentFrame();
+    await browser.switchToFrame(
+      await $('[src="/sitecore/shell/default.aspx?xmlcontrol=Workbox&mo=preview"]')
+    );
+    const jssDevelopmentWorkflowCheckbox = await (await $('label=JSS Development Workflow')).$(
+      '..//input'
+    );
+    if (!(await jssDevelopmentWorkflowCheckbox.getAttribute('checked'))) {
+      await browser.click({ element: await $('label=JSS Development Workflow') });
+    }
+    const developmentModeCollapsed = await $$('[class="scSectionCenter"]');
+    const contentModeCollapsed = await $$('[class="scSectionCenter"]');
+
+    const numOfDevItems = await (await developmentModeCollapsed[0]).getText();
+    console.log(numOfDevItems);
+    if (numOfDevItems.match(/\(([^)]+)\ /)) {
```

Comment:
Not sure there's anything necessarily to do, but this `publishContentItems` code is a bit hard to read (nested awaits, regexes, bit of arrow anti-pattern). Might be difficult for me to make any changes to it with confidence :)